### DISCUSSION TITLE
LFX Mentorship 1:1 online meet (Week 1)

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#136] [Mentorship]
If you have any problem to solve. Request 1:1 Google Meet.

# Link for Google Calendar (LXF Mentorship)
Anyone who has problem to solve the issue. Make 1:1 reservation on Google Calendar. There are lots of slot.
[Google Calendar](https://calendar.google.com/calendar/u/0?cid=MTdlYzU2MjFjNDlhN2Y3ZGFlMzc1NWJhM2EwNjhkZmNhNDA2NTY0OGMxYTM4ZGZmMWE3MWViZjlmMGIwZDU1NUBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)

# How to use
## Open shared calendar
Open the calendar. There are lots of slot. Reserve slot which has no name.
![image](https://github.com/cloudforet-io/cloudforet/assets/3993088/6c1afb5b-8ec3-4505-98ea-a0b9d149df0e)

## Add Comment in this thread.
I will update the Google Calendar title and invite you.
Mentor's timezone KST(Korea Standard Time), please check with your timezone.

**Comment by choonho** [comment]
Please comment. I will invite you.

**Reply by SammyBloom** [reply]
Wednesday, 8-9 am (WAT)

**Reply by SammyBloom** [reply]
Wednesday 1-2 pm (WAT). Thank you.

**Reply by Sumeiyah** [reply]
Wednesday 10-11 (EAT)
Testing environment Windows 16gb, 512gb

**Reply by choonho** [reply]
What is your current status?
Which step do you have problem?

**Reply by Asfawh** [reply]
Dec 13, 2023 Wednesday 1-2 (PST). I have some list of question step 1 and step 2. 
I wish there is some sort of instance message like slack. Thank you

**Reply by choonho** [reply]
@Asfawh I think you mean 1pm (PST). It is 6am (KST). My proposal is
1) 5pm(PST), 10am(KST)
2) 6pm(PST), 11am(KST)

Sorry! Slack is very good for instance message. But TSC decided to maintain github discussion.

**Reply by choonho** [reply]
What is your environment? Linux or Windows? I highly recommend Linux. Many person has hard time in Windows.

**Reply by Asfawh** [reply]
@choonho both time you propose works for me specially 6pm(PST), 11am(KST) and my environment is Linux

**Reply by choonho** [reply]
@Asfawh [1:1] Habtamu Asfaw Google Meet 6pm(PST),  https://meet.google.com/ane-hwfb-ejw

**Comment by choonho** [comment]
> Wednesday, 8-9 am (WAT)

Reserved Samuel, I also invited by gmail.

What is your testing environment? 
* Ubuntu? Mac? Windows?
* CPU and Memory size

**Reply by SammyBloom** [reply]
Ubuntu
16GB, 512GB.

Thank you so much

**Comment by Cvr421** [comment]
Hello @choonho  Can I join today meeting 
My environment is window 
Ram : 8 GB
Memory size : 256 SSD and 1TB Hard disk 
Processor : intel i5 10 GEN

**Reply by choonho** [reply]
@Cvr421 Yes. Anyone can join the meeting. Open google meet URL.

**Reply by Cvr421** [reply]
The meeting will start at 7 pm in indian time. Is this right

**Reply by Cvr421** [reply]
@choonho  I also want to contribute to code base . Can you guide me from were should i start . OR Any good first issue .

**Comment by Gaurav122000** [comment]
Hello @choonho Can I join today's meeting
My environment is also a window
Ram: 16 GB
Memory size: 1 tb SSD
Processor: Ryzen 5 3600

**Reply by Gaurav122000** [reply]
@choonho Which time you available

**Reply by choonho** [reply]
Hello. Please check the calendar. There are time slot which title has anyone.

**Reply by Gaurav122000** [reply]
there are no slots for today or tomorrow.

**Reply by choonho** [reply]
Today slot is already reserved. But if you want to see how to trouble shoot, it is OK to join Meeting.

**Reply by Gaurav122000** [reply]
Ok then I'll join and can you give me the Wednesday slot from 1-2 am

**Reply by AKACHI-4** [reply]
Hey @choonho, will there be more slots available after this week? I am okay with a slot after 8 PM on wed, thurs, fri,..

**Reply by choonho** [reply]
Yes I will create new

**Reply by AKACHI-4** [reply]
ohkay .. yah thanks then . actually I am getting system specific error ... I think my actual pc specs are low and that might be issue .. I am working on that .. but would love to join 1:1 with you..

**Reply by Gaurav122000** [reply]
@choonho Please reserve a slot for me on Wednesday from 1-2 am

**Reply by choonho** [reply]
@Gaurav122000 What is your timezone? I am confusing without your timezone?

**Reply by choonho** [reply]
@Gaurav122000 How about Friday 7:30pm (IST), 11:00pm(KST)

**Reply by Gaurav122000** [reply]
@choonho friday 7:30 done

**Reply by choonho** [reply]
@Gaurav122000 Which step did you have problem? If you succeed until **Step 6. Set the Helm Values and Upgrade the Chart**. I don't know how to port-forwarding in Windows.

**Reply by Gaurav122000** [reply]
@choonho I'm at the last step getting 404 error

**Reply by Gaurav122000** [reply]
@choonho I'm in the meeting Please join

**Reply by Asfawh** [reply]
I have the same 404 error at the last step when try to access/connect to the console, how did you fix this issues?

**Reply by choonho** [reply]
@Asfawh Many person has problem at last step. If every POD is running, there problem is port-forwarding such as lack of resource or unstable with minikube. Let's move on next step. I will check you finished Task #1.

**Comment by Cvr421** [comment]
@choonho when the slot is free so i can join . if we can do tomorrow that will be great . otherwise  wise give me any slot

**Reply by choonho** [reply]
@Cvr421 Let's make another time-slot. If you have problem step 7. It may be security-group issue or kubectl port-forwarding failure.

**Reply by Cvr421** [reply]
> @Cvr421 Let's make another time-slot. If you have problem step 7. It may be security-group issue or kubectl port-forwarding failure.

Ok @choonho please make the slot on today Or tomorrow at anytime.

**Reply by Cvr421** [reply]
@choonho can you fix my slot on tomorrow at 6:30 pm according to IST (Indian standard  time)

**Reply by choonho** [reply]
Sorry for late response. Can you contact me by linkedin message.

**Reply by Cvr421** [reply]
ok @choonho

**Comment by AKACHI-4** [comment]
@choonho I think thursday 6:30 PM to 7:30 PM will be okay for me .. kindly reserve that slot for me.. 
os -  window
ram - 8 GB
memory size : 256 SSD and 1TB Hard disk
processor : AMD Ryzen 3

**Reply by choonho** [reply]
@AKACHI-4 I reserved Thursday 6:30PM(IST), 10:00PM(KST) for you.

**Reply by AKACHI-4** [reply]
Yah @choonho sir Thanks 👍

**Reply by choonho** [reply]
@AKACHI-4 Which step did you have problem? If you succeed until Step 6. Set the Helm Values and Upgrade the Chart. I don't know how to port-forwarding in Windows.

**Reply by AKACHI-4** [reply]
![IMG_20231207_113836_906.jpg](https://github.com/cloudforet-io/cloudforet/assets/99159580/61d8b0e3-67ee-4f78-b988-63881f690e9d)


I m getting `TLS handshake timeout` actually, on gpt i found that it's error related to kubectl but not sure .. i didn't fix it .. will try

**Reply by choonho** [reply]
@AKACHI-4 Did you install WSL2 and Docker? 
From the log file, there is no docker engine.
Check https://minikube.sigs.k8s.io/docs/tutorials/wsl_docker_driver/

**Reply by AKACHI-4** [reply]
Yah I did .. it's lagging on start.. but fine after sometime

**Reply by AKACHI-4** [reply]
Hey @choonho .. sry but can can we schedule meet for some other day actually I have some interview preparation going to do. ..

about error so I will try it by myself I think some more research is needed though..

**Reply by choonho** [reply]
@AKACHI-4 Yes. Let's re-schedule after your interview.

**Reply by AKACHI-4** [reply]
@choonho Yah thanks.. i will check that if it's docker issue or other ..

**Comment by 0xPunkItachi** [comment]
I think Thursday 8:30-9:30 pm (IST) will be okay for me... Kindly reserve that slot for me... Sir @choonho 
Oss window
Ram-16gb
Memory 512 ssd
Processor ryzen 7

**Reply by choonho** [reply]
@YashCommits I reserved Thursday 8:30pm (IST), Friday 12:00am(KST)

**Reply by 0xPunkItachi** [reply]
Done sir 👍

**Reply by choonho** [reply]
@YashCommits Which step did you have problem? If you succeed until Step 6. Set the Helm Values and Upgrade the Chart. I don't know how to port-forwarding in Windows.

**Reply by 0xPunkItachi** [reply]
got it sir i was in 6th step and set the helm values and upgraded the chart ,still the crashloopbackoff error didnt left me alone... anyways sir i joined todays meet of 1:1 session 

![Screenshot (166)](https://github.com/cloudforet-io/cloudforet/assets/140940344/8c090d42-0815-49b8-944a-60be8d9b9c8d)

but i guess you might be busy today or maybe bcz of someother reason you are unable to join... and its perfectly fine sir i understand... dont worry i will try to install it by myself... will borrow a mac of one of my friend and try to install  in it, and if i get any error then will ask the folks who have succesfully installed it in mac

**Comment by DivvSaxena** [comment]
I think Friday 10-11pm (IST) will work for me... is the slot available for it @choonho ?

**Reply by choonho** [reply]
@DivvSaxena 10pm(IST)==1:30am(KST) is too late for me. How about 8:30pm(IST)==12:00am(KST)?

**Reply by DivvSaxena** [reply]
Ok , will work for me.

**Reply by choonho** [reply]
@DivvSaxena Which step did you have problem? If you succeed until Step 6. Set the Helm Values and Upgrade the Chart. I don't know how to port-forwarding in Windows.

**Reply by DivvSaxena** [reply]
hey @choonho would'nt be able to join today due to some urgent work!

**Reply by choonho** [reply]
Yes. OK. Let's make another day.

**Reply by Tyagiquamar** [reply]
@choonho Hi, Can you help me, I am free according to IST 7:30pm

**Reply by Suraj-kumar00** [reply]
Hello sir, How are you doing?
Sir can you please help me to locally setup this I reached to the 4th step here and now in the next steps I'm facing issues and errors. I'm doing it in WSL with Ubuntu and let know when your are free sir?

![WhatsApp Image 2023-12-20 at 20 02 06_ba175e64](https://github.com/cloudforet-io/cloudforet/assets/123288511/0c538c82-b542-4182-9239-8bdf3f408f82)

**Reply by choonho** [reply]
@Suraj-kumar00 open the url in your browser.  Save the content by manually.
